services/extraction-service/README.md

```
@@ -0,0 +1,77 @@
+```json
+{
+  "html": "<div class='item'>Content</div>",
+  "selector": ".item",
+  "extract_text": true,
+  "extract_html": false,
+  "extract_attributes": ["class", "id"]
+}
+```
+
+### XPath Extraction
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)